exporter/opentelemetry-exporter-prometheus-remote-write/setup.cfg

```diff
@@ -39,6 +39,8 @@ package_dir=
     =src
 packages=find_namespace:
 install_requires =
+    protobuf >= 3.13.0
+    requests == 2.25.0
     opentelemetry-api == 0.16.dev0
```

Comment:
Could you update these dependencies to 0.17.dev0 as well as the current version? It's causing some build failures.

Reply:
Updated